shared/src/style.css

```
@@ -0,0 +1,1025 @@
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: background-color 0.2s ease-in-out;
+  outline: 0;
+}
+.splitter [data-part="splitter"][data-focus] {
+  background: #b0baf1;
+}
+.splitter [data-part="splitter"]:active {
+  background: #3f51b5;
+  color: white;
+}
+
+.splitter [data-part="splitter"][data-disabled] {
+  opacity: 0.5;
+}
+.splitter .splitter-bar {
+  background-color: currentColor;
+}
+
+.splitter [data-orientation="horizontal"] .splitter-bar {
+  width: 2px;
+  height: 40px;
+}
+
+.splitter [data-orientation="vertical"] .splitter-bar {
+  height: 2px;
+  width: 40px;
+}
+
+.tags-input [data-part="control"] {
+  padding: 0 2px;
+  background: #fff;
+  border: 1px solid #ccc;
+  width: 40em;
+  border-radius: 2px;
+  box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.1);
+}
+
+.tags-input [data-part="control"][data-disabled] {
+  background: #f9f9f9;
+}
+
+.tags-input [data-part="control"][data-focus],
+.tags-input [data-part="control"]:focus {
+  border-color: red;
+  outline: 0;
+}
+
+.tags-input [data-part="tag"] {
+  background: #eee;
+  color: #444;
+  padding: 0 4px;
+  margin: 2px;
+  border: 1px solid #ccc;
+  border-radius: 2px;
+  font: inherit;
+  user-select: none;
+  cursor: pointer;
+  display: inline-block;
+}
+
+.tags-input [data-part="tag"][hidden] {
+  display: none !important;
+}
+.tags-input [data-part="tag"][data-selected] {
+  background-color: #777;
+  border-color: #777;
+  color: #eee;
+}
+.tags-input [data-part="tag"][data-disabled] {
+  opacity: 0.6;
+  cursor: default;
+}
+
+.tags-input [data-part*="input"] {
+  appearance: none;
+  padding: 3px;
+  margin: 0;
+  background: none;
+  border: none;
+  box-shadow: none;
+  font: inherit;
+  font-size: 100%;
+  outline: none;
+  display: inline-block !important;
+}
+
+.tags-input [data-part*="input"][hidden] {
+  display: none !important;
+}
+
+.tags-input [data-part*="input"]:disabled {
+  opacity: 0.6;
+}
+
+.tags-input [data-part="delete-button"] {
+  all: unset;
+}
+
+.pin-input [data-part="root"] {
+  width: 300px;
+  display: flex;
+  margin-bottom: 12px;
+  gap: 12px;
+}
+
+.pin-input [data-part="input"] {
+  width: 48px;
+  height: 48px;
+  text-align: center;
+  font-size: 24px;
+}
+
+@keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes fadeOut {
+  from {
+    opacity: 1;
+  }
+  to {
+    opacity: 0;
+  }
+}
+
+@keyframes spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes shrink {
+  from {
+    transform: scaleX(1);
+  }
+  to {
+    transform: scaleX(0);
+  }
+}
+
+.toast-group [data-part="root"] {
+  background: rgb(116, 116, 116);
+  border-radius: 4px;
+  color: white;
+  padding: 8px 8px 8px 16px;
+  width: 400px;
+  animation-fill-mode: forwards;
+  animation-duration: 0.2s;
+  animation-name: fadeIn;
+}
+
+.toast-group [data-part="root"]:not([data-open]) {
+  animation-duration: var(--remove-delay);
```

Let's scope this to the toast
```suggestion
  animation-duration: var(--toast-remove-delay);
```